ee/clickhouse/queries/session_recordings/test/test_clickhouse_session_recording_list.py

```diff
@@ -20,4 +24,61 @@ def _create_session_recording_event(**kwargs):
 
 
 class TestClickhouseSessionRecordingsList(ClickhouseTestMixin, factory_session_recordings_list_test(ClickhouseSessionRecordingList, _create_event, _create_session_recording_event, Action.objects.create, ActionStep.objects.create)):  # type: ignore
```

Comment:
Request: We've recently added a decorator called @snapshot_clickhouse_queries which will... snapshot clickhouse queries. Mind using that for the tests here? It will make reviewing easier.

Reply:
This is cool! Really useful